definitions/assertions/staging/assert_wave_contains_columns.sqlx

```diff
@@ -0,0 +1,78 @@
+config {
+    type: "assertion",
+    description: "Asserts that src_bmg_all_waves has the expected columns"
+}
```

Comment:
Sorry, just noticed that this won't run automatically unless we put a dependency on something. perhaps it is dependent on the src or first staging table?